fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/view/ValueElement.java

```diff
@@ -105,7 +105,7 @@ public int semanticHashCode() {
     }
 
     @Override
-    public int planHash() {
+    public int planHash(PlanHashKind hashKind) {
         return 0;
```

Comment:
for me to make better, this is the new stuff. you can keep the `Value...` code like this for now.